pubmed_parser/pubmed_oa_parser.py

```diff
@@ -264,11 +264,14 @@ def parse_pubmed_references(path):
             ref = reference.find("mixed-citation")
         elif reference.find("element-citation") is not None:
             ref = reference.find("element-citation")
+        elif reference.find("citation") is not None:
+            ref = reference.find("citation")
         else:
```

Comment:
This elif complex is highly inefficient. It gets every element twice. Also, we don't need to check whether an element `is None`, we can just evaluate it.

Reply:
- I implemented a function that iterates over the tags and returns the reference if something was found
- I get a [warning](https://github.com/lxml/lxml/blob/5770449be5e5f52df87d105aef335520f805aa06/src/lxml/etree.pyx#L2548) if a evaluate the element